# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../superkaramba/python24.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./acinclude.m4.orig	2005-02-16 01:24:12.000000000 -0300
+++ ./acinclude.m4	2005-02-16 01:26:33.408668752 -0300
@@ -4067,8 +4067,8 @@
 AC_DEFUN(KDE_CHECK_PYTHON,
 [
 
-
-  KDE_CHECK_PYTHON_INTERN("2.3", 
+ KDE_CHECK_PYTHON_INTERN("2.4",
+ [KDE_CHECK_PYTHON_INTERN("2.3", 
    [KDE_CHECK_PYTHON_INTERN("2.2", 
      [KDE_CHECK_PYTHON_INTERN("2.1", 
        [KDE_CHECK_PYTHON_INTERN("2.0", 
@@ -4076,6 +4076,7 @@
        ])
      ])
    ])
+  ])
 
 #  KDE_CHECK_PYTHON_INTERN("2.2", 
 #    [KDE_CHECK_PYTHON_INTERN("2.1", 
